es_test_data.py

```diff
@@ -70,16 +71,26 @@ def get_data_for_format(format):
     field_name = split_f[0]
     field_type = split_f[1]
 
+    if field_type == "bool":
+        return_val = random.choice([True, False])
+
     if field_type == "str":
         min = 3 if len(split_f) < 3 else int(split_f[2])
         max = min + 7 if len(split_f) < 4 else int(split_f[3])
-        length = random.randrange(min, max)
+        if min == max:
```

Comment:
Can you move this into a separate function? Looks like it's used in a bunch of places now.